runtime/test/strategies/create-views-tests.js

```diff
@@ -41,78 +41,78 @@ describe('CreateViews', function() {
     }
   };
 
-  it('Single reader', async () => {
+  it('doesnt work with a single reader', async () => {
     await testManifest(`
       recipe
         ? as view
         Reader
           foo <- view
-    `, /* expectedToAssignFate = */ false);
+    `, false);
```

Comment:
you're OK not having names for these?

Reply:
I suppose an alternative would be two different functions instead of testManifest:

```
await assertAssignsFate(`
    ...
```

Reply:
I wasn't thrilled about adding a variable, but I like the second suggestion, added the named functions.